custom_dc/env.list.sample

```diff
@@ -12,9 +12,10 @@ MAPS_API_KEY=
 # Local paths must be absolute paths.
 INPUT_DIR=
 # Output directory in Google Cloud Storage (GCS) or local drive for generated csv files.
-# GCS paths must be of the form gs://<BUCKET>/<FOLDER>.
+# GCS paths must be of the form gs://<BUCKET>/<FOLDER>. If you specify a GCS path, be 
+# sure to set USE_CLOUDSQL=true.
```

Comment:
They need to set more than USE_CLOUDSQL. So how about: If you specify a GCS path, you need to use Cloud SQL.

Reply:
Good point, I will actually point to relevant parameters section.